CMakeLists.txt

```diff
@@ -388,14 +388,6 @@ if(BUILD_TESTS)
       ${CMAKE_CURRENT_SOURCE_DIR}/src/node/rpc/test/tx_status_test.cpp
     )
 
-    # TODO: Port or remove entirely? add_unit_test( member_voting_test
```

Comment:
Is this now entirely covered by existing end-to-end tests?

Reply:
Yes, checked in #2476

Reply:
Ah! I'd missed that. Awesome news!